moto/emr/models.py

```diff
@@ -2,16 +2,21 @@
 from datetime import datetime
 from datetime import timedelta
 
+import warnings
+
 import pytz
 from boto3 import Session
 from dateutil.parser import parse as dtparse
 from moto.core import BaseBackend, BaseModel
+from moto.ec2 import ec2_backends
```

Comment:
Can we move the import inside the `run_job_flow`-method?
The EC2-service has some dependencies that are not required for EMR, so users should only have to install them when calling the `run_job_flow`